composer.json

```diff
@@ -70,13 +70,16 @@
         "process-timeout": 1800,
         "fxp-asset": {
            "enabled": false
-        },
-        "platform": {"php": "7.0"}
```

Comment:
That should be updated to what is actually on the server.

Reply:
OK. Let's wait for https://github.com/yiisoft-contrib/yiiframework.com/issues/960 to be resolved.